src/mrs/views.py

```diff
@@ -146,11 +148,45 @@ def get(self, request, *args, **kwargs):
         return response
 
 
-class NotFoundView(generic.View):
+def bad_request_view(request, exception):
```

Comment:
Pour rester homogène, ne pourrait-on pas rester full class-based-views ?

Reply:
Non, les vues d'erreur Django doivent accepter request/exception et ne peuvent donc pas être class based. Sauf si j'ai loupé quelque chose !

Reply:
Oui mais en python, on a [plein de moyens de faire des callables](https://github.com/django/django/blob/master/django/views/generic/base.py#L48-L81) ;)

> A view is a callable which takes a request and returns a response. This can be more than just a function, and Django provides an example of some classes which can be used as views. These allow you to structure your views and reuse code by harnessing inheritance and mixins.
[Introduction Class Based Views](https://docs.djangoproject.com/en/2.2/topics/class-based-views/)

En effet, les CBV sont des callables a partir du moment ou on appelle as_view() qui retourne une fonction:

```
In [3]: generic.View.as_view()                                                                                                                                                
Out[3]: <function django.views.generic.base.View(self, request, *args, **kwargs)>
```

Voici un petit crash course:

```
In [13]: class ExcView(generic.View): 
    ...:     def get(self, request, exc, *args, **kwargs): 
    ...:         print('exc:', exc, 'args:', args, 'kwargs:', kwargs) 
    ...:                                                                                                                                                                      

In [14]: from django.test.client import RequestFactory                                                                                                                        

In [15]: ExcView.as_view()(RequestFactory().get('/'), 'My Exception !')                                                                                                       
exc: My Exception ! args: () kwargs: {}
```

Suite dans le commentaire suivant ...